src/main/java/graphql/execution/SubscriptionExecutionStrategy.java

```diff
@@ -125,4 +158,10 @@ private ExecutionStrategyParameters firstFieldOfSubscriptionSelection(ExecutionS
         return parameters.transform(builder -> builder.field(firstField).path(fieldPath));
     }
 
+    private ExecutionStepInfo createSubscribedFieldStepInfo(ExecutionContext executionContext, ExecutionStrategyParameters parameters) {
+        Field field = parameters.getField().getSingleField();
+        GraphQLObjectType parentType = (GraphQLObjectType) parameters.getExecutionStepInfo().getUnwrappedNonNullType();
+        GraphQLFieldDefinition fieldDef = getFieldDef(executionContext.getGraphQLSchema(), parentType, field);
+        return createExecutionStepInfo(executionContext, parameters, fieldDef, parentType);
+    }
```

Comment:
This is weird but needed.  We want a execution step info here for the instrumentation - the soon to be called completeValue call will do something similar soon but we need this to descrive the "beginSubscribedFieldEvent" in instrumentation terms